admin/app/controllers/authenticated/organizations/list.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  queryParams: ['pageNumber', 'pageSize', 'name', 'type', 'code'],
+  pageNumber: 1,
+  pageSize: 10,
+  name: null,
+  type: null,
+  code: null,
+
+  actions: {
+    triggerFiltering(fieldName, value) {
+      this.set(fieldName, value);
```

Comment:
Il manque un debounce sur cette action pour éviter d'envoyer une requête à chaque fois

Reply:
oui c'est prévu, d'où le `development in progress` ;)

Reply:
aaaah mon mauvais !
Je reviendrai plus tard :)